### PR TITLE
docs: addition of param and result sections

### PR DESCRIPTION
**Pull request by hachreak**
* Improves documentation. (addresses #22)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: invenio_admin/ext.py [hunk 1]
```diff
@@ -39,15 +39,28 @@ class _AdminState(object):
     """State for Invenio-Admin."""
 
     def __init__(self, app, admin, permission_factory, view_class_factory):
-        """Initialize state."""
+        """Initialize state.
+
+        :param app: The Flask application.
+        :param admin: The Flask-Admin application.
+        :param permission_factory: The permission factory to restrict the
```
Comment (alizeepace) [inline invenio_admin/ext.py]: to restrict access

File: invenio_admin/ext.py [hunk 2]
```diff
@@ -92,7 +108,18 @@ def init_app(self,
                  view_class_factory=protected_adminview_factory,
                  index_view_class=AdminIndexView,
                  **kwargs):
-        """Flask application initialization."""
+        """Flask application initialization.
+
+        :param app: The Flask application.
+        :param entry_point_group: The entry point group lo load extensions.
```
Comment (alizeepace) [inline invenio_admin/ext.py]: typo? "lo"

File: invenio_admin/permissions.py [hunk 3]
```diff
@@ -28,10 +28,15 @@
...
 
 action_admin_access = ActionNeed('admin-access')
+"""Define the action needed by the default permission factory."""
 
 
 def admin_permission_factory(admin_view):
-    """Factory for creating a permission for an admin.
+    """Default factory for creating a permission for an admin.
+
+    It tries to load A :class:`invenio_access.permissions.DynamicPermission`
```
Comment (alizeepace) [inline invenio_admin/permissions.py]: is the capital A wanted?

File: invenio_admin/permissions.py [hunk 4]
```diff
@@ -28,10 +28,15 @@
...
+    if `invenio_access` is installed.
+    Otherwise, it load to load a :class:`flask_principal.Permission`.
```
Comment (alizeepace) [inline invenio_admin/permissions.py]: it loads/it tries to load

**Comment by jirikuncar** [comment]
- [ ] https://github.com/inveniosoftware/invenio-admin/pull/28/files#diff-05287a5c299bff612c62140bd2f722a5R4 update copyright year
- [ ] add yourself to AUTHORS file

**Comment by hachreak** [comment]
ping @jirikuncar

**Pull request merged**